### PR TITLE
fixes mem leak from PROPVARIANT

### PR DESCRIPTION
**Pull request by KillerBOSS2019**
This is pretty noticeable memory leak from `CreateDevice` that gets data from store `PROPVARIANT` aka the `value = store.GetValue(pk)`. This PR will use ole32 PropVariantClear to clear `value` to solve the memory leak issue.

### CODE REVIEW

File: pycaw/api/mmdeviceapi/depend/structures.py [hunk 1]
```diff
@@ -1,10 +1,9 @@
-from ctypes import Structure, Union
+from ctypes import Structure, Union, byref, windll
 from ctypes.wintypes import DWORD, LONG, LPWSTR, ULARGE_INTEGER, VARIANT_BOOL, WORD
 
 from comtypes import GUID
 from comtypes.automation import VARTYPE, VT_BOOL, VT_CLSID, VT_LPWSTR, VT_UI4
 
-
```
Comment (AndreMiras) [inline pycaw/api/mmdeviceapi/depend/structures.py:7]: Linter complaining here
```
pycaw/api\mmdeviceapi\depend\structures.py:7:1: E302 expected 2 blank lines, found 1
```

File: pycaw/api/mmdeviceapi/depend/structures.py [hunk 2]
```diff
@@ -39,6 +38,9 @@ def GetValue(self):
             return
         else:
             return "%s:?" % (vt)
+
```
Comment (AndreMiras) [inline pycaw/api/mmdeviceapi/depend/structures.py:41]:
```
pycaw/api\mmdeviceapi\depend\structures.py:41:1: W293 blank line contains whitespace
```

**Review by AndreMiras — CHANGES_REQUESTED** [review]
Good catch thanks!
Could you address the linter issues before we can merge?
https://github.com/AndreMiras/pycaw/actions/runs/4633151053/jobs/8203821389?pr=72

File: pycaw/utils.py [hunk 3]
```diff
@@ -246,6 +246,7 @@ def CreateDevice(dev):
                     continue
                 # TODO
                 # PropVariantClear(byref(value))
+                value.clear()
```
Comment (AndreMiras) [inline pycaw/utils.py:249]: Oh can you also remove the TODO since this got addressed

**Comment by KillerBOSS2019** [comment]
Thanks!

**Review by AndreMiras — APPROVED** [review]
Awesome thanks :rocket:

**Pull request merged**